### PR TITLE
Delete profile id during file share snapshot creation.

### PR DESCRIPTION
**Pull request by jackhaibo**
**What this PR does / why we need it**:
Delete profile id option during file share snapshot creation.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
#905

### CODE REVIEW

File: install/devsds/lib/opensds.sh [hunk 1]
```diff
@@ -84,7 +84,7 @@ osds::opensds::install(){
     export OPENSDS_AUTH_STRATEGY=$OPENSDS_AUTH_STRATEGY
     export OPENSDS_ENDPOINT=http://localhost:50040
     build/out/bin/osdsctl profile create '{"name": "default", "description": "default policy", "storageType": "block"}'
-    build/out/bin/osdsctl profile create '{"name": "default", "description": "default policy", "storageType": "file", "provisioningProperties":{"ioConnectivity": {"accessProtocol": "NFS"},"DataStorage":{"StorageAccessCapability":["Read","Write","Execute"]}}}'
+    build/out/bin/osdsctl profile create '{"name": "default", "description": "default policy", "storageType": "file", "provisioningProperties":{"ioConnectivity": {"accessProtocol": "nfs"},"DataStorage":{"StorageAccessCapability":["Read","Write","Execute"]}}}'
```
Comment (leonwanghui) [inline install/devsds/lib/opensds.sh]: Can u confirm if it's `NFS` or `nfs`?

File: osdsctl/cli/filesharesnapshot.go [hunk 2]
```diff
@@ -97,7 +96,6 @@ func init() {
 
 	fileShareSnapshotCreateCommand.Flags().StringVarP(&shareSnapName, "name", "n", "", "the name of the fileshare snapshot")
 	fileShareSnapshotCreateCommand.Flags().StringVarP(&shareSnapDesp, "description", "d", "", "the description of the fileshare snapshot")
-	fileShareSnapshotCreateCommand.Flags().StringVarP(&shareSnapProfileID, "profileId", "P", "", "the description of the fileshare snapshot")
```
Comment (leonwanghui) [inline osdsctl/cli/filesharesnapshot.go]: I don't get it why you remove this field.

File: install/devsds/lib/opensds.sh (hunk 1 above)
Comment (Shruthi-1MN) [reply install/devsds/lib/opensds.sh]: It is not required now, source code will take to create accessprotocol default as 'nfs', if you don't provide these values. Better remove accessprotocol and storageaccesscapability. It is done in PR#883

File: osdsctl/cli/filesharesnapshot.go (hunk 2 above)
Comment (jackhaibo) [reply osdsctl/cli/filesharesnapshot.go]: See issue #905

**Comment by leonwanghui** [comment]
Please resolve the ci problem.

File: install/devsds/lib/opensds.sh (hunk 1 above)
Comment (jackhaibo) [reply install/devsds/lib/opensds.sh]: Ok, I will delete it.

File: install/devsds/lib/opensds.sh [hunk 3]
```diff
@@ -84,7 +84,7 @@ osds::opensds::install(){
     export OPENSDS_AUTH_STRATEGY=$OPENSDS_AUTH_STRATEGY
     export OPENSDS_ENDPOINT=http://localhost:50040
     build/out/bin/osdsctl profile create '{"name": "default", "description": "default policy", "storageType": "block"}'
-    build/out/bin/osdsctl profile create '{"name": "default", "description": "default policy", "storageType": "file", "provisioningProperties":{"ioConnectivity": {"accessProtocol": "NFS"},"DataStorage":{"StorageAccessCapability":["Read","Write","Execute"]}}}'
+    build/out/bin/osdsctl profile create '{"name": "default", "description": "default policy", "storageType": "file", "provisioningProperties":{"ioConnectivity": {},"DataStorage":{}}}'
```
Comment (leonwanghui) [inline install/devsds/lib/opensds.sh]: Personally I prefer to add the accessProtocol and accessCapabilities explicitly.

Comment (jackhaibo) [reply install/devsds/lib/opensds.sh]: Ask @PravinRanjan10

**Pull request closed without merging**